bundled-docker/openmrs/Dockerfile

```diff
@@ -1,5 +1,9 @@
 FROM openmrs/openmrs-core:2.7.4-amazoncorretto-17
 
+ENV OMRS_EXTRA_LABONFHIR_KEYSTOREPASS="27e3af2b790009d6b8a2fa4eb6c9d8600a92215fa119ff2139e8b29a1bee72a5" \
+    OMRS_EXTRA_LABONFHIR_TRUSTSTOREPASS="27e3af2b790009d6b8a2fa4eb6c9d8600a92215fa119ff2139e8b29a1bee72a5" \
+    OMRS_EXTRA_LABONFHIR_PASSOWORD="admin"
```

Comment:
At what point are these env vars loaded?

Reply:
When the container is created from this docker image.